src/backup.py

```diff
@@ -766,18 +766,35 @@ def backup(self):
         """Apply backup methods"""
 
         for method in self.methods:
-            logger.debug(
-                m18n.n(
-                    "backup_applying_method_" + method.method_name, method=method.method
+            try:
+                logger.debug(
+                    m18n.n(
+                        "backup_applying_method_" + method.method_name,
+                        method=method.method,
+                    )
+                )
+            except AttributeError:
+                logger.debug(
+                    m18n.n(
+                        "backup_applying_method_" + method.method_name,
+                        method=method.method_name,
```

Comment:
eeeh it's really not obvious from reading the code what's the difference between these two blocks ... can't we use `if hasattr()` instead ?

Reply:
eg. `method=method.method if hasattr(method, "method") else method.method_name` ?

(Im not even sure to understand why method needs to be passed to the string considering the string key is already method-specific?)